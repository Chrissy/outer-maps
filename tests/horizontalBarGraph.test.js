import React from "react";
import HorizontalBarGraph from "../components/horizontalBarGraph.js";
import renderer from "react-test-renderer";

it("renders", () => {
  const tree = renderer
    .create(
      <HorizontalBarGraph
        keys={["cat", "dog", "bird"]}
        values={[50, 500, 5000]}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
