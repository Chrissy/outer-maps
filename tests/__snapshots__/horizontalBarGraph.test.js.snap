// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    className="bar"
  >
    <div
      className="barColor"
      style={
        Object {
          "width": "1%",
        }
      }
    />
    <div
      className="barLabel"
    >
      cat
    </div>
  </div>
  <div
    className="bar"
  >
    <div
      className="barColor"
      style={
        Object {
          "width": "10%",
        }
      }
    />
    <div
      className="barLabel"
    >
      dog
    </div>
  </div>
  <div
    className="bar"
  >
    <div
      className="barColor"
      style={
        Object {
          "width": "100%",
        }
      }
    />
    <div
      className="barLabel"
    >
      bird
    </div>
  </div>
</div>
`;
